internal/handler/getPathParameter.go

```diff
@@ -0,0 +1,19 @@
+package handler
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/gin-gonic/gin"
+)
+
+func GetPathParameter(c *gin.Context, parameter string) (uint, bool) {
```

Comment:
I would favor using `param/Param` everywhere.

Reply:
I favor consistency and I don't care much either way but feel free to submit a PR